recipes/libntv2/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conan import ConanFile
+from conan.tools.files import get, copy
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class LibNtv2Conan(ConanFile):
+    name = "libntv2"
+    description = "Library to create, dump, and use NTv2 files"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Esri/ntv2-file-routines/"
+    topics = ("ntv2", "geospatial", "datum")
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "no_mutexes": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "no_mutexes": False,
+    }
+
+    def export_sources(self):
+        copy(self, "CMakeLists.txt", self.recipe_folder, self.export_sources_folder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            self.options.rm_safe("fPIC")
+
+        self.settings.rm_safe("compiler.libcxx")
+        self.settings.rm_safe("compiler.cppstd")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.variables["NTV2_NO_MUTEXES"] = self.options.no_mutexes
+        tc.variables["NTV2_ROOT_DIR"] = self.source_folder.replace("\\", "/")
```

Comment:
```suggestion
        tc.cache_variables["NTV2_NO_MUTEXES"] = self.options.no_mutexes
        tc.cache_variables["NTV2_ROOT_DIR"] = self.source_folder.replace("\\", "/")
```
Since these options are defined using `option()` in CMakeLists.txt, which creates a cache variable, wouldn't `tc.cache_variables` be more correct? Please someone correct me if I'm wrong.